src/balance.rs

```diff
@@ -25,7 +25,7 @@ pub async fn cli(wallet_path: &Path, balance: &Balance) -> Result<()> {
     let wallet = load_wallet(wallet_path)?;
     let mut addresses = read_cached_addresses(&wallet.crypto.ciphertext)?;
     if !balance.account.unverified.unverified {
-        let prompt = "Please enter your password to verify accounts: ";
+        let prompt = "wallet password to verify accounts: ";
```

Comment:
Optional nit: this is the only place "Please enter your" is omitted

Reply:
ah, I must have highlighted and pasted over it by mistake. I'll add it back in!